src/Npgsql/PostgresDatabaseInfo.cs

```diff
@@ -61,8 +61,16 @@ class PostgresDatabaseInfo : NpgsqlDatabaseInfo
     /// </summary>
     public bool IsRedshift { get; private set; }
 
+    /// <summary>
+    /// True if the backend is YugabyteDB; otherwise, false.
+    /// </summary>
+    public bool IsYugabyteDB { get; set; }
+
     /// <inheritdoc />
-    public override bool SupportsUnlisten => Version.IsGreaterOrEqual(6, 4) && !IsRedshift;
+    public override bool SupportsUnlisten => Version.IsGreaterOrEqual(6, 4) && !IsRedshift && !IsYugabyteDB ;
```

Comment:
Shall we just comment out the upstream code and set this value as `false`?

Reply:
Done